lib/edit-site-export.php

```diff
@@ -29,7 +29,7 @@ function gutenberg_edit_site_export() {
 	$template_query = new WP_Query(
 		array(
 			'post_type'      => 'wp_template',
-			'post_status'    => array( 'publish', 'auto-draft' ),
+			'post_status'    => array( 'publish' ),
```

Comment:
I believe "publish" is the default status for the query, so I think we can remove this line. (see https://developer.wordpress.org/reference/classes/WP_Query/#status-parameters)

Reply:
For the lazy ones 😄 

> post_status (string / array) – use post status. Retrieves posts by post status. Default value is ‘publish‘, but if the user is logged in, ‘private‘ is added. Public custom post statuses are also included by default. And if the query is run in an admin context (administration area or AJAX call), protected statuses are added too. By default protected statuses are ‘future‘, ‘draft‘ and ‘pending‘.

According to the description, the default for logged-in admins is: `publish`, `private`, `future`, `draft`, `pending`. I'd keep this as `array( 'publish' )`

Reply:
Thanks for the clarification :)